src/modules/portfolio/ui/components/Footer.tsx

```
@@ -18,7 +18,7 @@ const Footer = () => {
 
         <ul className='flex items-center gap-10 justify mt-4 sm:mt-0'>
           <li><a target='_blank' href='https://github.com/lFGrahaml'>Github</a></li>
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)